src/main/resources/config/application-artemis.yml

```diff
@@ -3,6 +3,9 @@
 # ===================================================================
 
 artemis:
+  iris:
```

Comment:
Is this required or did you push this by accident?

Reply:
accident, thanks Kaan